Rakefile

```diff
@@ -166,6 +166,19 @@ task :c2m_check_version_all_dirs, [:last_checked_b4_date, :profile] => [:environ
   $stdout.flush
 end
 
+desc "Fire off C2M for a single druid"
+task :c2m_check_druid, [:last_checked_b4_date, :druid] => [:environment] do |_t, args|
+  begin
+    last_checked = args[:last_checked_b4_date].to_s
+    druid = args[:druid].to_sym
+    CatalogToMoab.check_version_druid(last_checked, druid)
+    puts "#{Time.now.utc.iso8601} Catalog to Moab version check on all roots is done."
```

Comment:
if you're going to have these puts statements, there should be one before, and one after, and they should have language appropriate to running a single druid.  "... on all roots is done" is a bit off base.